rust/hyperlane-base/src/types/local_storage.rs

```diff
@@ -100,6 +106,19 @@ impl CheckpointSyncer for LocalStorage {
         Ok(())
     }
 
+    async fn write_operator_registration(
```

Comment:
I see what you mean about room to dedupe - imo this is fine, don't worry too much about duping here, especially if we're trying to just get stuff out the door